internal/xds/bootstrap/bootstrap.yaml.tpl

```diff
@@ -8,6 +8,23 @@ admin:
     socket_address:
       address: {{ .AdminServer.Address }}
       port_value: {{ .AdminServer.Port }}
+{{- if .StatsMatcher  }}
+stats_config:
+  stats_matcher:
+    inclusion_list:
+      patterns:
+      {{- range $a, $s := .StatsMatcher.Prefixs }}
```

Comment:
can remove `$a` or use `_`?
use item instead of `s`

Reply:
done